app/mailers/expert_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class ExpertMailer < ApplicationMailer
+  helper MailerHelper
+  layout 'mailers/layout'
+
+  def send_dossier_decision(avis_id)
+    @dossier = Avis.find(avis_id).dossier
+    @avis = Avis.find(avis_id)
```

Comment:
On peut essayer de récupérer l'avis et le dossier en une seule requête :

```suggestion
    @avis = Avis.eager_load(:dossier).find(avis_id)
    @dossier = @avis.dossier
```